internal/services/compute/shared_image_version_resource.go

```diff
@@ -237,6 +286,9 @@ func resourceSharedImageVersionCreate(d *pluginsdk.ResourceData, meta interface{
 				AllowDeletionOfReplicatedLocations: utils.Bool(d.Get("deletion_of_replicated_locations_enabled").(bool)),
 			},
 			StorageProfile: galleryimageversions.GalleryImageVersionStorageProfile{},
+			SecurityProfile: &galleryimageversions.ImageVersionSecurityProfile{
```

Comment:
if `uefi_settings` is not configured, `SecurityProfile` should not be set a value?

Reply:
I thought a bit about it... As nothing else is using `ImageVersionSecurityProfile` right now, you're right, we have to initialize it a bit broader.
It will be `SecurityProfile: &galleryimageversions.ImageVersionSecurityProfile{},` now and filled if `uefi_settings` are set.